src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -637,7 +631,7 @@ public class LibraryServiceTest {
     mockLibraryService.setResponses(expectedResponses);
 
     String formattedName = LibraryServiceApi.formatBookName("[SHELF]", "[BOOK]");
-    String indexName = "indexName746962392";
+    String indexName = "default index";
```

Comment:
I think it makes sense to make initial value override the value generator so we could set values when necessary (next_page_token is an use case). The best way I think is to change the generator to not load from the sample_code_init fields.